zboxcore/sdk/allocation.go

```diff
@@ -446,7 +446,7 @@ func (a *Allocation) uploadOrUpdateFile(localpath string,
 	}
 	thumbnailSize := int64(0)
 	if len(thumbnailpath) > 0 {
-		fileInfo, err := os.Stat(thumbnailpath)
+		fileInfo, err := sys.Files.Stat(thumbnailpath)
```

Comment:
Is there any difference between the `sys.Files.Stat` and the `os.Stat`, if no, let's just use the os.Stat to keep it simple and avoid further reading about the new package.

Reply:
os.Stat doesn't work on WebAssembly. Because chrome can't read local file system. that is why it should be bridged on sys.FS